tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/sound.py

```diff
@@ -5,11 +5,19 @@
 from marionette_driver import expected, By, Wait
 
 from gaiatest.apps.base import Base
+from gaiatest.apps.ring_tone.app import RingTone
 
 
 class Sound(Base):
+    _page_locator = (By.ID, 'sound')
     _ring_tone_selector_locator = (By.CSS_SELECTOR, '.ring-tone-selection')
     _current_ring_tone_locator = (By.CSS_SELECTOR, '.ring-tone-selection > small')
+    _alert_selector_locator = (By.ID, 'alerts')
+    _manage_tones_selector_locator = (By.ID, 'manage-tones')
+
+    @property
+    def screen_element(self):
```

Comment:
This is used to scroll the current page.  Since the scroll method (inside gaiagraphictestcase.py) needs the element, I thought this is better than to add scroll methods for all of the subviews of settings.